api/api_sources/sources/database/database-schema/schema-sqls/AspectCodeSchema.sql

```diff
@@ -0,0 +1,39 @@
+-- ### Creating Table: observation_aspect_code ### --
+
+        
+CREATE TABLE observation_aspect_code ();
+ALTER TABLE observation_aspect_code ADD COLUMN observation_aspect_code_id SERIAL PRIMARY KEY;
+ALTER TABLE observation_aspect_code ADD COLUMN observation_aspect_code VARCHAR(3) NOT NULL UNIQUE;
+ALTER TABLE observation_aspect_code ADD COLUMN description VARCHAR(100) NULL;
+ALTER TABLE observation_aspect_code ADD COLUMN active_ind BOOLEAN NOT NULL DEFAULT TRUE;
+
+
+        
+-- ### Creating Comments on table ### --
+
+        
+COMMENT ON TABLE observation_aspect_code IS 'Observation area aspect code. Aspect code provides integer enum encoded values for directional aspect of the observed species. The typical values like North facing or south facing';
```

Comment:
Maybe update the description here too... to reflect that that aspect code reflects the direction in which the slope faces. And need to update the reference to "integer enum values".